qa/tasks/ceph_test_case.py

```diff
@@ -18,6 +18,7 @@ class CephTestCase(unittest.TestCase):
     # Environment references
     mounts = None
     fs = None
+    recovery_fs = None
```

Comment:
That intuitively makes more sense to me, too, but I preferred putting it next to the primary filesystem.